desk/app/grouper.hoon

```diff
@@ -149,26 +166,34 @@
         :-  joiner.bite
         [id %add memo [%notice ~] ~]
       =/  =cage  chat-dm-action+!>(`action:dm:c`action)
-      =*  dez  %^  tell:log  `token.bite  %info
+      =*  dez  %^  lure-log  %info  'DM Invite Sent'
                ~[leaf+"{<joiner.bite>} invited to DM"]
       (snoc [dez caz] [%pass wir %agent dock %poke cage])
     ?~  group=(~(get by fields.metadata.bite) 'group')
-      ~[(tell:log `token.bite %warn 'group field missing' ~)]
+      :_  ~
+      %^  lure-log  %warn  'Group Field Missing'
```

Comment:
how are you thinking about `%warn` vs `%crit` here?

Reply:
The bite here is still going to be processed and a DM invite is going to be send regardless (unless some other error happens) – I would not think it is possible for the group field to be missing from metadata if the original Lure invite was for a group.

It just occurs to me that there used to be personal lure invites – do we still support those? If so, perhaps this should not trigger any warning, as a personal Lure invite would not contain `group` field in the metadata.

Reply:
We use a stub group ID for personal invites because if the `group` field wasn't structured like a flag, the DM invite wasn't getting sent. Those will always be `~zod/personal-invite`

I think we'll end up getting a _Group Invite Failed_ log for those, but we can deal with that noise in PostHog for now. Ideally eventually we'd special case that `group` and not log a failure.